docs/faq/README.md

```diff
@@ -52,14 +52,19 @@ System Cron:
 * Check if the **oc_jobs** table has a **reserved_at** entry with a value other than 0. If it does for whatever reason, set it to 0. You can check this by executing:
 
   ```sql
-  SELECT reserved_at FROM oc_jobs WHERE (argument = '["OCA\\News\\Cron\\Updater","run"]' OR class = 'OCA\\News\\Cron\\Updater');
+  SELECT reserved_at FROM oc_jobs WHERE (argument = '["OCA\News\Cron\Updater","run"]' OR class = 'OCA\News\Cron\Updater');
```

Comment:
The double `\\` are required since the `\` is treated as an escape character.

Reply:
If I past the original command to my SQL workplace (Eclipse + DBeaver) it won't work.

Reply:
Interesting, `SELECT reserved_at FROM oc_jobs WHERE (argument = '["OCA\\News\\Cron\\Updater","run"]' OR class = 'OCA\\News\\Cron\\Updater' OR class = 'OCA\\News\\Cron\\UpdaterJob')` works for me (MySQL + PhpMyAdmin), apparently Postgres handles it different.

Reply:
I have modified the SQL statement and set it to a more generic version which should run at PostgreSQL as well as MySQL/MariaDB.

Could you please verify that if you're using MySQL/MariaDB?